tap_hubspot/__init__.py

```diff
@@ -307,8 +320,37 @@ def lift_properties_and_versions(record):
             record['properties_versions'] += versions
     return record
 
+def post_search_endpoint(url, data, params=None):
+    params = params or {}
+    hapikey = CONFIG['hapikey']
+    if hapikey is None:
+        if CONFIG['token_expires'] is None or CONFIG['token_expires'] < datetime.datetime.utcnow():
+            acquire_access_token_from_refresh_token()
```

Comment:
So this method fills the access token into the global CONFIG variable? Any reason not to return the access token? Looks weird to me that you call a method here and it's not entirely clear what it's doing. This isn't a big deal...

Reply:
This is how the other request function did it, so we just copied it in hopes that it wouldn't be too confusing.

> So this method fills the access token into the global CONFIG variable?

Yes, exactly. And the global `CONFIG` gets checked before the tap makes any request

Reply:
ok